packages/css-library/src/stylesheets/modules/m-maintenance-banner.scss

```diff
@@ -17,7 +17,7 @@
     }
 
     .usa-alert-heading {
-      font-family: "Source Sans Pro";
+      font-family: $font-family-sans;
```

Comment:
This file is already importing the variables file, and `$font-farmily-sans` contains the updated `Source Sans Pro Web` naming convention. Renaming this fixes rendered font.